feat(discharge): redesigned discharge + chores

Mxrxk opened the pull request:
### Changes
- Separated Hv from Lv
- Separated Hv+ and Hv- into 2 separate connectors
- Fixed Isolator footprint
- Renamed "discharge-breakout-pcb" to "Discharge"

### Related Issue(s)
Closes #100 

### Checklist
- [x] ERC passes with no errors.
- [x] DRC passes with no errors.
- [x] No files which should not be under version control added (see [README](https://github.com/sufst/pcb/blob/main/README.md)).
- [x] Library paths are relative paths.

### Additional Notes
There seems to be an issue in the DRC about the pads for the relay. Not sure what this is about.
There are also warnings about the Footprints not matching libraries for footprints with 3D models. Resolving them causes 3D models to disappear.

Commit:
feat(Discharge): Redesigned Discharge pcb + chores

t-bre commented on Discharge/Discharge.kicad_pcb:
The silkscreen needs a clear marking of what side is TS and what side is LV, e.g. with the HV sense: 

<img width="754" alt="Screenshot 2023-04-29 at 18 52 23" src="https://user-images.githubusercontent.com/31595761/235316999-67247a4f-b151-46af-aed2-30b80ca225a9.png">

t-bre replied:
Silkscreen here looks misaligned

<img width="284" alt="Screenshot 2023-04-29 at 18 53 10" src="https://user-images.githubusercontent.com/31595761/235317028-a130c8f0-ef63-4657-872f-e7551301e76f.png">

t-bre replied:
The GND trace spends a lot of time within that isolation area which I think might not be looked upon favourably at scrut. I'd be inclined to route it through the LV area first and then just have a small part of trace into the isolation area where absolutely necessary. 

<img width="293" alt="Screenshot 2023-04-29 at 18 54 11" src="https://user-images.githubusercontent.com/31595761/235317074-af392ce4-a6c1-4ce7-b4e9-a5bbc78b83ad.png">

t-bre replied:
The first DRC error indicates that the component was created as SMD type in the SUFST library, when it should be through hole. 

<img width="818" alt="Screenshot 2023-04-29 at 18 56 15" src="https://user-images.githubusercontent.com/31595761/235317182-af8a84b6-18f5-47b2-9ff6-899546dc1cf0.png">

You can fix this in the footprint properties:

<img width="444" alt="Screenshot 2023-04-29 at 18 57 34" src="https://user-images.githubusercontent.com/31595761/235317239-d723ccd1-4923-4ba0-8ccc-885819c99cc2.png">

The rest of the DRC errors are a thing that's new to KiCad 7 - if you add a 3D model to a footprint, it considers the footprint as having changed from the library. I guessed either the default severity for this has been changed from "ignore" to "warning" or this is just a bug. Can be fixed by changing the severity to "ignore" in the miscellaneous design rules (in the board setup window):

<img width="687" alt="Screenshot 2023-04-29 at 18 59 23" src="https://user-images.githubusercontent.com/31595761/235317316-ece93fc8-4587-4b30-9793-f7f4515d2aed.png">

t-bre reviewed (changes requested):
Small bits to change but otherwise looks good

Commit:
fix(Discharge): final fixes

Mxrxk commented:
Changes:
- Labelled sides as "TS" and "LV"
- Fixed silkscreen miss alignment
- Shortened the amount of GND trace within isolation area
- Fixed relay footprint issues
- Changed the DRC check "Footprint doesn't match copy in library" to "ignore"
- Changed version from v1.0.2 to v1.1.0
- Edited sheet layout information

t-bre committed:
fix(discharge): minor tweaks to silkscreen

also typos on sheet info

t-bre reviewed (commented):
Looks good.

Made a small tweak to the silkscreen to replace "HV+" and "HV-" with "TS+" and "TS-", but committed that fix myself.
